lib/cdc/util/relational_event.go

```diff
@@ -111,13 +113,16 @@ func (s *SchemaEventPayload) GetData(pkMap map[string]any, tc *kafkalib.TopicCon
 		retMap[constants.DatabaseUpdatedColumnMarker] = s.GetExecutionTime().Format(ext.ISO8601)
 	}
 
-	// Iterate over the schema and identify if there are any fields that require extra care.
-	afterSchemaObject := s.Schema.GetSchemaFromLabel(cdc.After)
-	if afterSchemaObject != nil {
-		for _, field := range afterSchemaObject.Fields {
+	return retMap
+}
+
+// parseValues will take `retMap` and `kind` (which part of the schema should we be inspecting) and then parse the values accordingly.
+// This will unpack any Debezium-specific values and convert them back into their original types.
+func (s *SchemaEventPayload) parseValues(retMap map[string]any, kind cdc.FieldLabelKind) map[string]any {
+	if schemaObject := s.Schema.GetSchemaFromLabel(kind); schemaObject != nil {
+		for _, field := range schemaObject.Fields {
```

Comment:
On the next line we have the value of the field, but we're assigning it to `_`, we could assign it to a variable and then we wouldn't have to do the `retMap[field.FieldName]` lookup again.

Reply:
True.